gauge-ruby.go

```diff
@@ -163,7 +163,7 @@ func main() {
 	}
 	if *start {
 		os.Chdir(projectRoot)
-		err = runCommand("ruby", "-e", "require 'gauge_runtime'")
+		err = runCommand("bundle", "exec", "ruby", "-e", "require 'gauge_runtime'")
```

Comment:
Why has this changed?

Reply:
When start runner for execution from command line we use `bundle exec gauge run` which make sure all your required gems as available for the project. Whiling starting the runner for LSP we need to do the same or else `gauge-ruby` gem will not be available. or users have to install `gauge-ruby` globally.